bee-protocol/src/protocol/protocol.rs

```diff
@@ -141,6 +155,7 @@ impl Protocol {
             PROTOCOL = Box::leak(protocol.into()) as *const _;
         }
 
+        Protocol::get().bus.add_listener(on_tip_candidate_found);
```

Comment:
why are you going through the `Bus` event system when can just directly get a reference to the `MsTangle`?

Reply:
To publish it to 3rd parties as well I guess

Reply:
True, I guess we don't need to expose this event to the 3rd parties (since it's not that interesting), so I will change that